doc/howto/json.md

```diff
@@ -0,0 +1,130 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Querying JSON files
+
+In this tutorial, we'll show you how to query JSON with JupySQL and [DuckDB](https://duckdb.org/docs/extensions/json.html).
+
+
+First, let's install the required dependencies:
+
+```{code-cell} ipython3
+%pip install jupysql duckdb duckdb-engine rich --quiet
+```
+
+Now, let's generate some data. Note that DuckDB expects your data to contain *one JSON object per line*; this format is called [JSON Lines](https://jsonlines.org/), and it often comes with the `.json`, `.jsonl.gz`, or `.jsonl.bz2` extension.
+
+Our sample data contains four rows:
+
+```{code-cell} ipython3
+from pathlib import Path
+import json
+
+data = [
+    {"name": "John", "age": 25,  "friends": ["Jake", "Kelly"], "likes": {"pizza": True}},
+    {"name": "Jake", "age": 20,  "friends": ["John"], "likes": {"pizza": False}},
+    {"name": "Kelly", "age": 21,  "friends": ["John", "Sam"], "likes": {"pizza": True}},
+    {"name": "Sam", "age": 22,  "friends": ["Kelly"], "likes": {"pizza": False}},
+]
+
+lines = ""
+
+for d in data:
+    lines += json.dumps(d) + "\n"
+
+Path("people.jsonl").write_text(lines)
+```
+
+```{code-cell} ipython3
+print(lines)
+```
+
+## Query
+
+Load the extension and start a DuckDB in-memory database:
+
+```{code-cell} ipython3
+%load_ext sql
+%sql duckdb://
+```
+
+Read the JSON data:
+
+```{code-cell} ipython3
+%%sql
+SELECT *
+FROM read_json_objects('people.jsonl')
+```
+
+## Extract fields
+
+Extract fields from each JSON record:
+
+```{code-cell} ipython3
+%%sql
+SELECT
+    json ->> '$.name' AS name,
```

Comment:
Is there a reference for the meaning of this syntax `json ->> $.name` which can be linked here?

Reply:
sure, added a link to the docs in the previous section